ibm/mas_devops/roles/suite_app_config/README.md

```diff
@@ -58,6 +58,19 @@ Defines the app components and versions to configure in the application workspac
 - Environment Variable: `MAS_APPWS_COMPONENTS`
 - Default: Application specific
 
+### mas_pod_templates_dir
+This role will look for a configuration files named:
+
+- `ibm-mas-manage-manageworkspace.yml`
+- `ibm-mas-manage-imagestitching.yml`
+- `ibm-mas-manage-slackproxy.yml`
+- `ibm-mas-manage-healthextworkspace.yml`
+
+The content of the configuration file should be the yaml block that you wish to be inserted into the ManageWorkspace CR. `ibm-mas-manage-manageworkspace.yml` will be inserted into the ManageWorkspace CR `spec -> podTemplates` whereas the component ones e.g, `ibm-mas-manage-imagestitching.yml` will be under `spec -> components -> civil -> podTemplates`. The ibm-mas-manage-ws operator will then pass this on to the corresponding component CR when available.
+
+This is an example of one of the components (civil) - refer to the [BestEfforts reference configuration in the MAS CLI](https://github.com/ibm-mas/cli/blob/master/image/cli/mascli/templates/pod-templates/best-effort/ibm-mas-manage-civil.yml).
```

Comment:
This link to the civil pod-templates doesn't exist and doesn't exist in the draft PR https://github.com/ibm-mas/cli/pull/931/files either. Is it going to be added or should it change?

Reply:
rename civil to correct name with imagestitching. File will be available once mas cli pr gets merged.